Lib/email/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2001-2007 Python Software Foundation
+# Copyright (C) 2001 Python Software Foundation
```

Comment:
Could we remove the copyright comments in files where the sole holder is the PSF? The general licence would cover this, I think?

Reply:
Me too
https://github.com/python/cpython/issues/126133#issuecomment-2447150671

Reply:
my personal preference is to not remove statements that exist.  just get us out of the continually sometimes updating the year heck. :)